src/Crud/Controller/BaseCrudController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Biig\Melodiia\Crud\Controller;
+
+use Biig\Melodiia\Bridge\Symfony\Response\FormErrorResponse;
+use Biig\Melodiia\Crud\CrudControllerInterface;
+use Biig\Melodiia\Response\ApiResponse;
+use Biig\Melodiia\Response\WrongDataInput;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Zend\Json\Exception\RuntimeException;
+use Zend\Json\Json;
+
+abstract class BaseCrudController implements CrudControllerInterface
+{
+    /**
+     * @return ApiResponse|FormInterface
+     */
+    protected function decodeInputData(FormFactoryInterface $formFactory, string $form, Request $request, bool $clearMissing = null)
+    {
+        if (null === $clearMissing) {
+            $clearMissing = true;
```

Comment:
`$clearMissing = !in_array($request->getMethod(), ['POST', 'PUT']);` would do the job

Reply:
Finally fixed!